### PR TITLE
test pr

### LINKED ISSUE #4: Chore: wording task

### PR DESCRIPTION
**Pull request by kiangkiangkiang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 Users can easily forecast stock price trends by simply providing an API key, making sophisticated financial analysis accessible and user-friendly.
 
-## Features
+## 🌟 Features
```
Comment (taochen1114) [inline README.md:30]: test comment 這個漂亮！

**Review by taochen1114 — COMMENTED** [review]
Nice Test

**Review by vic4code — COMMENTED** [review]
Awesome

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,3 @@ ignore*
 .vscode
 *.egg*
 data/*
-123
```
Comment (ShemYu) [inline .gitignore:7]: well done

**Comment by rshiue** [comment]
add default content of `.gitignore` for python please. 😃 

refer to GitHub guide: 
https://github.com/github/gitignore/blob/main/Python.gitignore

**Pull request closed without merging**